setup.cfg

```diff
@@ -36,7 +36,7 @@ zip_safe = True
 install_requires =
     six >=1.9
     swiglpk >=5.0.8
-    sympy >=1.0
+    sympy >=1.11.1
```

Comment:
Why not go with version 1.12 here?

Reply:
Would make optlang uninstallable with Python 3.7. I can add this if you're okay with dropping 3.7.

Reply:
Hmm, 3.7 has end of life on June 27, so it's not unreasonable to drop it, I think.

Reply:
Yep, sounds good. I'll set it to 1.12.

Reply:
Alternatively, we make one last release that pins sympy to 1.11.1 which will be the last version to support 3.7. Then immediately release a next version with sympy >= 1.12 that drops support.